content/blog/mapping-kubernetes-services-to-hpe-ezmeral-container-platform-gateway.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Mapping Kubernetes Services to HPE Ezmeral Runtime Gateway
+date: 2021-10-26T05:44:07.981Z
+author: Cenz Wong
+authorimage: https://avatars.githubusercontent.com/u/44856918?s=400&u=37bb095377cd6b4ad21c3a7ab8b5afe185a46941&v=4
+tags:
+  - hpe-ezmeral-runtime
+---
+
+**Editor’s Note – NAME CHANGE: HPE Ezmeral Container Platform is now HPE Ezmeral Runtime.** HPE Ezmeral Runtime is a software platform designed to deploy cloud-native and non-cloud-native applications using 100% open source Kubernetes – running on bare-metal or virtualized infrastructure – on-premises, on any cloud, and at the edge. For more information on why the name was changed, please click [here](https://community.hpe.com/t5/HPE-Ezmeral-Uncut/HPE-Ezmeral-Container-Platform-is-now-HPE-Ezmeral-Runtime/ba-p/7151720#.YW7nOxrMKM8).
+
+[Introduction]
+
+HPE Ezmeral Runtime (formerly known as HPE Ezmeral Container Platform) comes with one or more gateway hosts. A gateway host acts as a proxy server that carries client requests like HPE Ezmeral Runtime UI, REST API calls, Kubernetes API and containerized application service endpoints. The Gateway host maps both the IP address of the Controller host and the private IP endpoints of services running on the Kubernetes nodes inside the Kubernetes clusters to publicly-accessible IP addresses/ports.
+

```

Comment:
Cenz, I'm learning how to use previews in social media to promote the blog posts. One of the things I've discovered is that a preview will pull up the very first image in your blog post. I was wondering if there was any way to put this image towards the top... making it the first image... since it would appear to be a good one to use to promote the post.

![image](https://user-images.githubusercontent.com/77496261/141339194-a454f46c-5862-494c-8abf-96ca7b6896c2.png)

What do you think? Or, do you think there's another image we could add at the very beginning of the post that you would prefer to have as the promotional picture?

Reply:
Thats good ideas! I believe a good looking preview would attract more people to click on the blog. One suggestion would be to standardize the dimension of the preview becoz it would give people a sense of professional. I recommend create one using https://www.canva.com/en_gb/ or just use PowerPoint to do that.